calliope/backend/pyomo/model.py

```diff
@@ -101,32 +102,23 @@ def generate_model(model_data):
 
     # Constraints
     constraints_to_add = [
-        'energy_balance.load_constraints',
-        'dispatch.load_constraints',
-        'network.load_constraints',
-        'costs.load_constraints',
-        'policy.load_constraints',
-        'milp.load_constraints',
-        'group.load_constraints'
+        i.split('.py')[0] for i in os.listdir(constraints.__path__[0])
+        if '__' not in i
     ]
 
-    if backend_model.__calliope_run_config['mode'] != 'operate':
-        constraints_to_add.append('capacity.load_constraints')
-
-    if hasattr(backend_model, 'loc_techs_conversion'):
-        constraints_to_add.append('conversion.load_constraints')
-
-    if hasattr(backend_model, 'loc_techs_conversion_plus'):
-        constraints_to_add.append('conversion_plus.load_constraints')
+    # The list is sorted to ensure that some constraints are added after pyomo
+    # expressions have been created in other constraint files.
+    # Ordering is given by the number assigned to the variable ORDER within each
```

Comment:
I feel a bit uneasy about having lots of files with the same value for ORDER. How about using increments of 10 and starting with 10 for energy_balance, 20 for dispatch, and so on in the order that they are currently loaded in, permitting enough space to hack stuff in between (a bit like logging levels perhaps)?

Reply:
I think it is good to be clear about what things need to be loaded in after other things. In reality, only a few files need to be loaded after others (because preceding files attach new objects to the backend model) and it is a lot more clear that those files are different if only they are loaded in later. But changing to increments of 10 makes a lot of sense.

Reply:
Ok, so we're moving this to increments of 10, right?

Reply:
Right